### PR TITLE
Streamlit fp

### PR DESCRIPTION
**Pull request by aarondr77**
# Description

- Adds a data apps foundational page. I think Data Apps is a better header than Streamlit since a lot of people I've talked to don't know what Streamlit is. 
- Adds streamlit to a bunch of other pages where relevant (homepage, plans, financial services, etc)
- Take a look at these apps: [Data Cleaning ](https://github.com/mito-ds/data-cleaning-demo/tree/data-cleaning-to-export) [Snowflake data analysis ](https://github.com/mito-ds/snowflake-streamlit-bank-performance-demo) too
**When we release the Streamlit component + make our streamlit apps public, we need to replace the links in the Streamlit Gallery!**

# Testing

- Mobile + Desktop website review
- Take a look at the Streamlit app repos and make any changes. @naterush merge the Data Cleaning PR if you're happy with with it 

# Documentation

Note if any new documentation needs to addressed or reviewed.

### CODE REVIEW

File: trymito.io/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -46,19 +46,10 @@ const Footer = (): JSX.Element => {
                         <Link href='/low-code-sql'>Low-Code SQL</Link>
                     </li>
                     <li className={classNames(footerStyle.nav_item)}>
-                        <Link href='/infrastructure-integration-python-tool'>Infrastructure Integration</Link>
+                        <Link href='/data-app'>Data App</Link>
```
Comment (naterush) [inline trymito.io/components/Footer/Footer.tsx:49]: This feels like a really meaningless word. Can we think of a better name?

File: trymito.io/pages/plans.tsx [hunk 2]
```diff
@@ -690,7 +698,7 @@ const Plans: NextPage = () => {
                           Feature <br/> settings
                         </h2>
                         <p>
-                          Customize Mito by toggling code optimization, auto-documentation, seleting between light and dark mode, and more. (coming soon)
+                          Customize Mito by toggling on/off onbaording tours, support locations, seleting between light and dark mode, and more.
```
Comment (naterush) [inline trymito.io/pages/plans.tsx:701]: onboarding

File: trymito.io/pages/data-app.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+
+            <section className={classNames(titleStyles.title_card)}>
+                <h1 className={titleStyles.title}>
+                    Upgrade Python scripts to interactive Streamlit Apps
```
Comment (naterush) [inline trymito.io/pages/data-app.tsx:36]: Turn Python Scripts into Data Apps

File: trymito.io/pages/data-app.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+                </h1>
+                <p className={titleStyles.description}>
+                    Turn one hour of automation savings into tens of hours by sharing automation scripts through Streamlit dashboards.
```
Comment (naterush) [inline trymito.io/pages/data-app.tsx:39]: This is kinda a pitch for streamlit, not really Mito in streamlit. Maybe that's fine.

File: trymito.io/pages/data-app.tsx [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+                </div>
+                <a href={DATA_VERIFICATION_STREAMLIT_APP_LINK} target="_blank" rel="noreferrer" className={pageStyles.link_with_p_tag_margins}>
+                    Try this app now →
+                </a>
+            </section>
+            <section className={textImageSplitStyles.text_image_section_container}>
+                <div className={pageStyles.subsection}>
+                    <div className={textImageSplitStyles.functionality_text}>
+                        <h1>
+                            From automation script, to <span className='text-highlight'>automation app</span>
```
Comment (naterush) [inline trymito.io/pages/data-app.tsx:58]: From Python script to `data app`

File: trymito.io/pages/data-app.tsx [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+                        </h1>
+                        <p>
+                            It only takes one data analyst to build a script for an entire team to automate their similar workflows.
```
Comment (naterush) [inline trymito.io/pages/data-app.tsx:61]: I think we should not talk about data analysts, but rather our user profile no?

File: trymito.io/pages/data-app.tsx [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+                <div className={pageStyles.subsection}>
+                    <div className={classNames(textImageSplitStyles.functionality_media, textImageSplitStyles.functionality_media_supress_bottom_margin, 'display-desktop-only-inline-block')}>
+                        <Image src={'/explore_data_visually.png'} alt='Explore data with Mito' width={500} height={250} layout='responsive'/>
+                    </div>
+                    <div className={textImageSplitStyles.functionality_text}>
+                        <h1>
+                            Make your data explorable to non-technical users
+                        </h1>
+                        <p> 
+                            Setting up database connections is confusing even to engineers.
```
Comment (naterush) [inline trymito.io/pages/data-app.tsx:81]: Finding the right dataset can be hard even for engineers.

File: trymito.io/pages/data-app.tsx [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+                </div>
+            </section>
+
+            <section>
+                <div className={classNames(pageStyles.subsection, pageStyles.subsection_column, 'center')}>
+                    <h2>
+                        Mito Streamlit Gallery
+                    </h2>
+                    <p>
+                        Find one you like, then use it to build your own
```
Comment (naterush) [inline trymito.io/pages/data-app.tsx:99]: Find one you like, and then take our code to build your own.

**Review by naterush — CHANGES_REQUESTED** [review]
A few rephrasing things. I like this page a lot. LGTM!

File: trymito.io/pages/data-app.tsx (hunk 6 above)
Comment (aarondr77) [reply trymito.io/pages/data-app.tsx:61]: ooo yeah good catch lol

File: trymito.io/pages/data-app.tsx (hunk 4 above)
Comment (aarondr77) [reply trymito.io/pages/data-app.tsx:39]: Yeah, I agree, but I think that's fine. Let's see how it goes

**Pull request merged**